### DISCUSSION TITLE
Improved weapon placement handling

### DISCUSSION THREAD

**Discussion by VaelophisNyx** [endless-sky/endless-sky#9309] [Ideas]
As someone who plays with mods that have ships with admittedly FAR too many turret mounts, I'd like to poke around and see if anyone could mock-up a better screen to handle weapon mount placements in game; currently if you have enough weapons your I menu becomes cumbersome, or borderline unsuable

![image](https://github.com/endless-sky/endless-sky/assets/1534478/095f1fc9-3ba8-49b0-bd88-0b6bda6d9b8a)
(This is 102 turrets and 26 gunports.)

It'd be nice if there was a more streamlined way to handle this, something less prone to menu overflowing

**Comment by bene-dictator** [comment]
See #7453

**Reply by VaelophisNyx** [reply]
oh, perfect
